guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc

```diff
@@ -0,0 +1,44 @@
+[id="Extracting_GPG_Public_Key_Fingerprints_from_a_Release_Files_{context}"]
+= Extracting GPG Public Key Fingerprints from a Release Files
+
+You can use GPG public keys the verify the authenticity of DEB repositories by verifying the signature of the `Release` file.
```

Comment:
DEB looks like an acronym. 

Would Debian not be better? I googled DEB repositories and just got Debian repositories. 
What is the standard?

Reply:
I chose `DEB` as a way of including Debian and Ubuntu repositories. Maybe `repositories containing DEB packages` is a good alternative?

Sometimes, people also use `APT`, but that's more of a standard application to download/install packages.

Reply:
Advanced Package Tool = APT - so this is a valid abbreviation.

However, DEB is not an abbreviation... and I don't find it as a standard reference in other Debian/Ubuntu resources. 
I think the DEB looks really bad and misleading, but I will leave it as your call.

Reply:
Changed to `APT`.